.github/workflows/bench-deploy.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Install criterion
         run: cargo install cargo-criterion
       - name: Run benchmarks
-        run: just --dotenv-filename bench.env gpu-bench fibonacci_lem
+        run: cd benches && just --dotenv-filename bench.env gpu-bench fibonacci_lem
```

Comment:
With a `run` step the best practice is to equip this with a `working-directory:` attribute.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory